Library/Formula/terraform-inventory.rb

```diff
@@ -0,0 +1,61 @@
+class TerraformInventory < Formula
+  desc "Terraform State → Ansible Dynamic Inventory"
+  homepage "https://github.com/adammck/terraform-inventory"
+  url "https://github.com/adammck/terraform-inventory/archive/v0.6.tar.gz"
+  sha256 "af09a2606db728fd046e0ce5a6960386e5f766f8d187eea15220764fd669187e"
+  head "https://github.com/adammck/terraform-inventory.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/stretchr/testify" do
+    url "https://github.com/stretchr/testify.git",
+    :revision => "1f4a1643a57e798696635ea4c126e9127adb7d3c"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    app_path = buildpath/"src/github.com/adammck/terraform-inventory"
+    app_path.install Dir["*"]
+
+    cd "src/github.com/adammck/terraform-inventory" do
+      system "go", "get"
```

Comment:
I did use gdm to generate the go_resource block. The dependency is only used to build the unit tests, so it feels like we should be able to avoid including it. But if the tests are built at the same time as the rest of the app with `go build`, then maybe not. What do you think?